src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -169,7 +173,7 @@ public void progress(ProgressMessage message) throws DockerException {
       ObjectMapperProvider.class,
       JacksonFeature.class,
       LogsResponseReader.class,
-      ProgressResponseReader.class);
+                       ProgressResponseReader.class);
```

Comment:
nit: There are still a bunch of random whitespace changes like here. Can we remove these?